src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -218,26 +221,178 @@ export async function buildMapView(id: string, dotNetReference: any, long: numbe
             addGraphic(graphicObject, id);
         })
 
+        setEventListeners(view, dotNetRef, eventRateLimitInMilliseconds, activeEventHandlers);
+        
+        unsetWaitCursor(id);
+    } catch (error) {
+        logError(error, id);
+    }
+}
+
+function setEventListeners(view: __esri.View, dotNetRef: any, eventRateLimit: number | null, 
```

Comment:
Does this only get called when the mapview is created? If yes, we are not supporting the scenario where you might connect and disconnect to events at different points in the app?